.github/workflows/azure-static-web-apps-deploy.yml

```diff
@@ -1,6 +1,7 @@
 name: Azure Static Web Apps web components
 
 on:
+  workflow_dispatch:
```

Comment:
manual dispatch don't work if workflow doesn't exist in default branch ( in our case master ) - let's remove it to avoid confusion

Reply:
fixing!